packages/cadl-rlc-test/test/playfab/cadl-output/src/playFab.ts

```diff
@@ -10,7 +10,6 @@ export default function createClient(
   options: ClientOptions = {}
 ): PlayFabClient {
   const baseUrl = options.baseUrl ?? `${endpoint}`;
-  options.apiVersion = options.apiVersion ?? "0000-00-00";
```

Comment:
why is this line missing ?

Reply:
I removed because `"0000-00-00"` seems to be a bug in the compiler when the CADL doesn't define a service version